karma.conf.js

```diff
@@ -26,7 +26,14 @@ module.exports = function (config) {
             ]
         },
 
-        reporters: [ 'mocha', 'coverage' ],
+        junitReporter: {
+            outputDir: 'junit/',
+            useBrowserName: false
+        },
+
+        reporters: [ 'mocha', 'coverage', 'junit', 'coveralls' ],
```

Comment:
ps we should probably copy the comment on "why junit" here, since there's no java code and it might confuse ppl